OpenRA.Mods.Common/Widgets/Logic/Editor/CommonSelectorLogic.cs

```diff
@@ -33,9 +34,22 @@ public abstract class CommonSelectorLogic : ChromeLogic
 		protected string[] allCategories;
 		protected string searchFilter;
 
-		public CommonSelectorLogic(Widget widget, World world, WorldRenderer worldRenderer, string templateListId, string previewTemplateId)
+		[TranslationReference]
+		static readonly string None = "none";
+
+		[TranslationReference]
+		static readonly string SearchResults = "search-results";
+
+		[TranslationReference]
+		static readonly string All = "all";
```

Comment:
These feel like very generic translation ids for what may be a quite specific use?

Reply:
This is on purpose to have it only translated once. I don't see a case where a language needs to translate this differently per context.